src/realm.js

```diff
@@ -343,7 +365,6 @@ export class Realm {
   wrapInGlobalEnv<T>(callback: () => T): T {
     let context = new ExecutionContext();
     context.isStrict = this.isStrict;
-    context.lexicalEnvironment = this.$GlobalEnv;
```

Comment:
In line 65 of prepack-node-environment.js we have a similar code pattern. Should that also be fixed up?